src/assets/scripts/aircraft/AircraftController.js

```diff
@@ -33,45 +33,33 @@ export default class AircraftController {
 
     /**
      * @for AircraftController
-     * @method aircraft_generate_callsign
-     * @param airlineName
+     * @method aircraft_auto_toggle
      */
-    aircraft_generate_callsign(airlineName) {
-        // TODO: this should live in the AirlineModel
-        const airline = window.airlineController.airline_get(airlineName);
-
-        if (!airline) {
-            console.warn(`Airline not found: ${airlineName}`);
-
-            return `airline-${airlineName}-not-found`;
-        }
-
-        return airline.generateFlightNumber();
+    aircraft_auto_toggle() {
+        prop.aircraft.auto.enabled = !this.aircraft.auto.enabled;
     }
 
     /**
      * @for AircraftController
-     * @method aircraft_auto_toggle
+     * @method isCallsignInList
+     * @param callsign {string}
+     * return {boolean}
      */
-    aircraft_auto_toggle() {
-        prop.aircraft.auto.enabled = !this.aircraft.auto.enabled;
+    isCallsignInList(callsign) {
+        return this.aircraft.callsigns.indexOf(callsign) !== -1;
```

Comment:
Oh, okay, I figured it was useful because it would also check an array so you could use the same function since the two concepts are similar. But I know what happens when I assume...